public/video-ui/src/slices/video.ts

```diff
@@ -0,0 +1,66 @@
+import { Video } from '../services/VideosApi';
+import { blankVideoData } from '../constants/blankVideoData';
+import { createSlice, Action, PayloadAction } from '@reduxjs/toolkit';
+
+const initialState: null | false | Video = null;
+
+interface VideoAction extends Action {
+  video?: Video;
+}
+
+const isVideoAction = (action: Action): action is VideoAction => {
+  return [
+    'VIDEO_GET_RECEIVE',
+    'VIDEO_CREATE_RECEIVE',
+    'VIDEO_SAVE_REQUEST',
+    'VIDEO_SAVE_RECEIVE'
```

Comment:
These are action types which also have significance for the saveState reducer. My current thinking is to split these actions into two, the `setVideo` action created in this PR and a `setSaveState` to be created  when we migrate the saveState reducer to RTK.